### PR TITLE
Databricks: Generic cloud platform

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/reverse-etl/databricks.mdx [hunk 1]
```diff
@@ -77,21 +73,19 @@ For more information on the data import settings, refer to the <a href="https://
 
 ### Where can I obtain the connection credentials for Databricks?
 
-To obtain the **Host**, **Path**, and **Port** number required to set up Databricks source in RudderStack, follow these steps:
+To obtain the **Host**, **Path**, and **Port** number, follow these steps in your Databricks account:
```
Comment (ameypv-rudder) [inline docs/reverse-etl/databricks.mdx:76]:
```suggestion
To obtain the **Host**, **Path**, and **Port** number, go to your Databricks account and follow these steps:
```

**Review by ameypv-rudder — APPROVED** [review]
LGTM

File: docs/reverse-etl/databricks.mdx [hunk 2]
```diff
@@ -8,10 +7,35 @@ description: Step-by-step guide to ingest data from your Databricks cluster into
...
+### Step 1: Add a user
+
+- Add a new user (for example, user@example.com) by following the steps in the [Databricks documentation](https://docs.databricks.com/administration-guide/users-groups/users.html#add-a-user).
+
+### Step 2: Creating the RudderStack schema and granting permissions
+
+1. Create a dedicated schema `_rudderstack`.
+
+```sql
+CREATE SCHEMA _rudderstack;
```
Comment (Vikas26021999) [inline docs/reverse-etl/databricks.mdx:25]: In the query, _rudderstack -> `_rudderstack`

File: docs/reverse-etl/databricks.mdx [hunk 3]
```diff
@@ -8,10 +7,35 @@ description: Step-by-step guide to ingest data from your Databricks cluster into
...
+```
+
+<div class="warningBlock">
+The <code class="inline-code">_rudderstack</code> schema is used by RudderStack for storing the state of each data sync. <strong>This name should not be changed</strong>.
 </div>
 
+2. Grant full access to the schema `_rudderstack` for the user created above.
+
+```sql
+GRANT ALL PRIVILEGES ON SCHEMA `_rudderstack` TO `user@example.com`
```
Comment (Vikas26021999) [inline docs/reverse-etl/databricks.mdx:35]: Minor : should we ask the user explicitly to replace user@example.com with the user he created before
Sometimes in the flow, users just copy SQL commands and run it. WDYT?

**Pull request merged**